presto-orc/src/test/java/com/facebook/presto/orc/AbstractTestOrcReader.java

```diff
@@ -493,6 +494,19 @@ public void testDwrfInvalidCheckpointsForStripeDictionary()
                         .collect(toList()));
     }
 
+    @Test
+    public void testExtraLargeColumnValue()
+            throws Exception
+    {
+
+        Random rnd = new Random(0);
+        String val = rnd.doubles(50_000).mapToObj(Double::toString).collect(Collectors.joining(","));
+        tester.assertRoundTrip(
```

Comment:
this test doesn't use testRoundTrip() because there is some bug with the hive reader reading the list value test (unrelated to encryption)